### DISCUSSION TITLE
Is there any way for me to deploy nicegui using uwsgi?

### DISCUSSION THREAD

**Discussion by MuuXB** [zauberzeug/nicegui#1769] [Q&A]
### Description

I know that uvicorn can be used to start, but I want to integrate nicegui into my system. My system is managed by uwsgi. Is there any way to make it compatible?

**Comment by rodja** [comment]
I don't think it is possible. See Stack Overflow https://stackoverflow.com/questions/77140731/how-to-use-uwsgi-to-run-fastapi-program.
